.github/workflows/release.yml

```
@@ -63,9 +63,60 @@ jobs:
       uses: actions/setup-go@v6
       with:
         go-version-file: go.mod
+    - name: Read prime registry secrets
+      uses: rancher-eio/read-vault-secrets@main
+      with:
+        secrets: |
+          secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials username |  PRIME-REGISTRY-USERNAME;
+          secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials password |  PRIME-REGISTRY-PASSWORD;
+          secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials registry |  PRIME-REGISTRY-REGISTRY;
     - name: Update manifests
       run: |
         make release RELEASE_TAG=${{ env.TAG }} REGISTRY=${{ env.GHCR_REGISTRY }}
+    - name: Package and publish release manifests as OCI artifacts
+      env:
+        ORAS_VERSION: 1.2.3
+        ARTIFACT_NAME: ${{ env.PRIME-REGISTRY-REGISTRY }}/rancher/cluster-api-provider-rke2-components
+        REGISTRY_PASSWORD: ${{ env.PRIME-REGISTRY-PASSWORD }}
+        REGISTRY_USERNAME: ${{ env.PRIME-REGISTRY-USERNAME }}
+        REGISTRY_HOST: ${{ env.PRIME-REGISTRY-REGISTRY }}
```

```suggestion
        ARTIFACT_NAME: ${{ env.PRIME_REGISTRY }}/rancher/cluster-api-provider-rke2-components
        REGISTRY_PASSWORD: ${{ env.PRIME_REGISTRY_PASSWORD }}
        REGISTRY_USERNAME: ${{ env.PRIME_REGISTRY_USERNAME }}
        REGISTRY_HOST: ${{ env.PRIME_REGISTRY }}
```
In line with my other suggestion.